frontend-ssr/components/tableOfContents/TableOfContents.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import styles from './TableOfContents.module.scss';
+
+interface State {
+    activeTitleChapter: string,
+}
+
+interface Props {
+    chapters: Chapters[],
+}
+
+interface Chapters {
+    title: string,
+    id: string,
+}
+
+class TableOfContents extends React.Component<Props, State> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      activeTitleChapter: null,
```

Comment:
This is still named wrong. `activeTitleChapter` sounds to me like  it's holding some type of title, but instead it;s holding ids. Let's rename to `activeChapterId`